create-creds

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python3
+import sys
+
+from helpers import generate_credential_tokens, require_secrets
+
+
+def main(creds_in, creds_out):
+    require_secrets('osftok', 'triggered-by')
+    generate_credential_tokens(creds_in, creds_out)
```

Comment:
what's the lifetime of this token, just wondering if it won't expire during long LmP build.